### PR TITLE
in-progress websocket connection

### PR DESCRIPTION
**Pull request by Donadams50**

### CODE REVIEW

File: client/src/Chat.jsx [hunk 1]
```diff
@@ -1,44 +1,148 @@
...
-          id
-          user
-        }
-      }`
-
-const Messages = ({user}) => {
-  const {data} = useQuery(GET_MESSAGES);
-  if(!data){
-    return null
+const POST_MESSAGES = gql`
```
Comment (Skrpk) [inline client/src/Chat.jsx]: `POST_MESSAGE`, not messageS

File: client/src/Chat.jsx [hunk 2]
```diff
@@ -1,44 +1,148 @@
...
+            label="Content"
+            value={state.content}
+            onChange={(evt) =>
+              stateSet({
+                ...state,
+                content: evt.target.value
+              })
+            }
+            onKeyUp={(evt) => {
+              if (evt.keyCode === 13) {
```
Comment (Skrpk) [inline client/src/Chat.jsx]: Magic number antipattern. Create a variable called `ENTER_KEY_CODE` or smth like that and assign to it 13 and use it

File: client/src/Chat.jsx [hunk 3]
```diff
@@ -1,44 +1,148 @@
...
+                onSend();
+              }
+            }}
+          />
+        </Col>
+        <Col xs={2}>
+          <Button onClick={() => onSend()}>Send</Button>
```
Comment (Skrpk) [inline client/src/Chat.jsx]: Can be simplified to `<Button onClick={onSend}>Send</Button>`

File: client/src/Chat.jsx [hunk 4]
```diff
@@ -1,44 +1,148 @@
...
+      </Row>
+    </Container>
+  );
+};
+
+export default () => (
+  <ApolloProvider client={client}>
+    <Chat />
+  </ApolloProvider>
+);
```
Comment (Skrpk) [inline client/src/Chat.jsx]: Check out this capture https://nimb.ws/9k1nUd
Is it ok for now?

File: client/src/Chat.jsx (hunk 1 above)
Comment (Donadams50) [reply client/src/Chat.jsx]: okay. i was following a tutorial on youtube , so i did exactly what the guy did. Will make the changes

File: client/src/Chat.jsx (hunk 4 above)
Comment (Donadams50) [reply client/src/Chat.jsx]: Not yet, its still work in progress. i will complete it tommorow

**Pull request merged**